ee-security/README.adoc

```diff
@@ -0,0 +1,121 @@
+include::../shared-doc/system-requirements.adoc[leveloffset=+1]
+// Use of {jbossHomeName}
+include::../shared-doc/use-of-jboss-home-name.adoc[leveloffset=+1]
+// Add the Authorized Application User
+include::../shared-doc/add-application-user.adoc[leveloffset=+1]
+// Back Up the {productName} Standalone Server Configuration
+include::../shared-doc/back-up-server-standalone-configuration.adoc[leveloffset=+1]
+// Start the {productName} Standalone Server
+include::../shared-doc/start-the-standalone-server.adoc[leveloffset=+1]
+
+[[configure_the_server]]
+== Configure the Server
+
+You configure the security domain by running JBoss CLI commands. For your convenience, this quickstart batches the commands into a `configure-elytron.cli` script provided in the root directory of this quickstart.
+
+. Before you begin, make sure you do the following:
+
+* xref:back_up_standalone_server_configuration[Back up the {productName} standalone server configuration] as described above.
+* xref:start_the_eap_standalone_server[Start the {productName} server with the standalone default profile] as described above.
+
+. Review the `configure-elytron.cli` file in the root of this quickstart directory. This script adds the configuration that enables Elytron security for the quickstart components. Comments in the script describe the purpose of each block of commands.
+. Open a new terminal, navigate to the root directory of this quickstart, and run the following command, replacing `__{jbossHomeName}__` with the path to your server:
++
+[source,subs="+quotes,attributes+",options="nowrap"]
+----
+$ __{jbossHomeName}__/bin/jboss-cli.sh --connect --file=configure-elytron.cli
+----
++
+NOTE: For Windows, use the `__{jbossHomeName}__\bin\jboss-cli.bat` script.
++
+
+You should see the following result when you run the script:
++
+[source,options="nowrap"]
+----
+The batch executed successfully
+process-state: reload-required
+----
+
+// Build and Deploy the Quickstart
+include::../shared-doc/build-and-deploy-the-quickstart.adoc[leveloffset=+1]
+
+== Access the Application
+
+The application will be running at the following URL http://localhost:8080/{artifactId}/secured.
+
+To see and manipulate the HTTP headers within the HTTP requests it is recommended to use a client like `curl` to invoke the servlet. 
+
+[source,options="nowrap"]
+----
+$ curl -v http://localhost:8080/ee-security/secured
+...
+< HTTP/1.1 401 Unauthorized
+< Connection: keep-alive
+< X-MESSAGE: Please resubmit the request with a username specified using the X-USERNAME and a password specified using the X-PASSWORD header.
+----
+
+This first request shows the client is being promted to authenticate, the `X-MESSAGE` header is providing additional information as to how the client can achieve this.
+
+The request can now be submitted with the previously added user.
+
+[source,options="nowrap"]
+----
+$ curl -v http://localhost:8080/ee-security/secured -H "X-Username:darranl" -H "X-Password:password1!"
```

Comment:
This line should be: 

$ curl -v http://localhost:8080/ee-security/secured -H "X-Username:quickstartUser" -H "X-Password:quickstartPwd1!"